apollo/interfaces/azure/function_app.py

```diff
@@ -143,3 +144,37 @@ def agent_api(req: func.HttpRequest, context: func.Context):
     Endpoint to execute sync operations.
     """
     return wsgi_middleware.handle(req, context)
+
+
+@app.function_name(name="cleanup_df_data")
+@app.schedule(
+    schedule="0 0 3,15 * * *", arg_name="timer", run_on_startup=False
+)  # run every day at 3 AM/PM
+@app.durable_client_input(client_name="client")
+async def cleanup_durable_functions_data(
+    timer: func.TimerRequest, client: DurableOrchestrationClient
+) -> None:
+    created_time_from = datetime.today() - timedelta(
+        days=365 * 10
+    )  # datetime.min or None not supported
+    created_time_to = datetime.today() - timedelta(days=1)

```

Comment:
should these times be timezone-aware? if the cron-schedule is UTC based, then we'll probably want these to be UTC as well.

Reply:
Good call, I think it's works fine with the local tz but I agree it's better to be timezone aware, made the changed and monitored the next execution, everything looks good. Thanks